src/spec/formatter.cr

```diff
@@ -62,14 +62,15 @@ module Spec
     end
   end
 
-  @@formatter = DotFormatter.new
+  @@formatters = [] of Formatter
+  @@formatters << DotFormatter.new
 
   # :nodoc:
-  def self.formatter=(@@formatter)
+  def self.formatters=(@@formatters)
```

Comment:
I think we wouldn't need this method anymore, and just leave `self.formatters`, and document it saying that this is the array of registered formatters, you can modify it before running the specs.